src/components/structures/LoggedInView.js

```diff
@@ -507,14 +558,16 @@ const LoggedInView = React.createClass({
             <div className='mx_MatrixChat_wrapper' aria-hidden={this.props.hideToSRUsers} onMouseDown={this._onMouseDown} onMouseUp={this._onMouseUp}>
                 { topBar }
                 <DragDropContext onDragEnd={this._onDragEnd}>
-                    <div className={bodyClasses}>
+                    <div ref={(div) => this.resizeContainer = div} className={bodyClasses}>
```

Comment:
LoggedInView probably is a class where we want to avoid passing anonymous functions here if possible as we don't want it to re-render unnecessarily.

Reply:
so basically do `ref={this.resizeContainer}`? Curious how this causes a re-render.

Reply:
i assume dave is thinking of the 'closures' section of https://engineering.musefind.com/our-best-practices-for-writing-react-components-dec3eb5c3fc8

Reply:
right, because each new instance of the function !== the previous, causing the subcomponent to rerender.